[debug] [2020-04-22T20:34:29.847Z] ----------------------------------------------------------------------
[debug] [2020-04-22T20:34:29.850Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-22T20:34:29.850Z] CLI Version:   8.1.1
[debug] [2020-04-22T20:34:29.850Z] Platform:      win32
[debug] [2020-04-22T20:34:29.851Z] Node Version:  v12.16.1
[debug] [2020-04-22T20:34:29.853Z] Time:          Wed Apr 22 2020 21:34:29 GMT+0100 (Irish Standard Time)
[debug] [2020-04-22T20:34:29.854Z] ----------------------------------------------------------------------
[debug] [2020-04-22T20:34:29.854Z] 
[debug] [2020-04-22T20:34:29.886Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-22T20:34:29.989Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-22T21:39:28.370Z] ----------------------------------------------------------------------
[debug] [2020-04-22T21:39:28.373Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-22T21:39:28.373Z] CLI Version:   8.1.1
[debug] [2020-04-22T21:39:28.373Z] Platform:      win32
[debug] [2020-04-22T21:39:28.374Z] Node Version:  v12.16.1
[debug] [2020-04-22T21:39:28.376Z] Time:          Wed Apr 22 2020 22:39:28 GMT+0100 (Irish Standard Time)
[debug] [2020-04-22T21:39:28.376Z] ----------------------------------------------------------------------
[debug] [2020-04-22T21:39:28.376Z] 
[debug] [2020-04-22T21:39:28.416Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-22T21:39:28.470Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2020-04-22T21:43:24.406Z] ----------------------------------------------------------------------
[debug] [2020-04-22T21:43:24.408Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-22T21:43:24.409Z] CLI Version:   8.1.1
[debug] [2020-04-22T21:43:24.409Z] Platform:      win32
[debug] [2020-04-22T21:43:24.409Z] Node Version:  v12.16.1
[debug] [2020-04-22T21:43:24.411Z] Time:          Wed Apr 22 2020 22:43:24 GMT+0100 (Irish Standard Time)
[debug] [2020-04-22T21:43:24.411Z] ----------------------------------------------------------------------
[debug] [2020-04-22T21:43:24.411Z] 
[debug] [2020-04-22T21:43:24.443Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-22T21:43:24.488Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\dunne\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
